src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
On review of the matplotlib documentation, it looks like *everything* in `matplotlib.pyplot` (i.e., `plt.*`) is intentionally stateful, and that `plt.subplots` modifies the current figure.  Why don't we just create an appropriate `matplotlib.axes.Axes` instance?